### PR TITLE
Gluggi variations

### LINKED ISSUE #10: Add view helper class, that separates all children from each other
This is useful if I want to list all variants of a single component.

Something along the lines of 
```css
.gluggi-previews * + *{
    margin-top: 30px;
}
```

### PR DESCRIPTION
**Pull request by apfelbox**
`.gluggi-variations` with a top margin of 25px (we previously used `2rem`, but with `25px` we are independent from rem definitions).

Fixes #10

### CODE REVIEW

**Comment by apfelbox** [comment]
Failing tests due to missing merge of #12. I will trigger a new build after #12 is merged.

**Comment by apfelbox** [comment]
Failing tests due to missing merge of #12. I will trigger a new build after #12 is merged.

File: kabafile.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+const kaba = require("kaba");
+
+const scss = kaba.shelf.scss({
+    input: "Resources/assets/scss"
```
Comment (cH40z-Lord) [inline kabafile.js]: Isn't that covered by the default kaba SCSS file glob?

Comment (apfelbox) [inline kabafile.js]: See: https://github.com/Becklyn/kaba

> Default value: `src/**/Resources/assets/scss/`

**Comment by cH40z-Lord** [comment]
LGTM :shipit:. Push the button

**Pull request merged**